proto/process/agent.proto

```diff
@@ -1892,3 +1912,44 @@ message NetworkPolicyPort {
 	int32 port = 2;
 	int32 endPort = 3;
 }
+
+message LimitRange {
```

Comment:
could you add a comment with a link to the original Kubernetes definition for reference?

Reply:
done 👍